src/scripts/heroAnimation/hero.js

```diff
@@ -119,19 +119,26 @@ function initHeader() {
   params.ctx = params.canvas.getContext("2d");
 
   const pointsLimiter = utils.calcPointsLimiter(params.width, params.height);
+  const gridSize = 100;
 
-  for (var x = 0; x < params.width; x = x + params.width / pointsLimiter) {
+  for (
+    var x = -gridSize;
+    x < params.width;
+    x = x + params.width / pointsLimiter
+  ) {
     for (var y = 0; y < params.height; y = y + params.height / pointsLimiter) {
-      var px = x + Math.random() * params.width / pointsLimiter;
-      var py = y + Math.random() * params.height / pointsLimiter;
-      var p = {
-        x: px,
-        originX: px,
-        y: py,
-        originY: py
-      };
-
-      params.points.push(p);
+      var px = utils.getRandomArbitrary(x, x + gridSize);
```

Comment:
do we have any reason to keep these vars in a wider scope? can we reduce their pollution scope using `let` or - even better `const` ?